interface/patient_file/summary/add_edit_issue.php

```diff
@@ -375,6 +375,7 @@ function ActiveIssueCodeRecycleFn($thispid2, $ISSUE_TYPES2) {
 <html>
 <head>
 <?php html_header_show();?>
+    <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-9-1/index.js"></script>
```

Comment:
I am note sure what you mean by out of syntax block. Just let me know if I am missing something.